Add docs for mint export (#4774)
mintlify/docs: deploy/export.mdx

---
title: "Offline export"
description: "Export your documentation site as a self-contained zip archive for offline viewing and distribution."
keywords: ["export", "offline", "zip", "static site", "distribution", "mint export"]
---

Use `mint export` to package your entire documentation site into a self-contained zip archive. Recipients can unzip and view the docs in their browser without an internet connection, a Mintlify account, or any build tools — they only need [Node.js](https://nodejs.org) installed.

> This is useful when you need to distribute documentation to users who can't access your live site, such as for on-premise customers, air-gapped environments, or internal compliance reviews.

Reviewer: ```suggestion
Use offline exports to distribute documentation to users who can't access your live site, such as for on-premise customers, air-gapped environments, or internal compliance reviews.
```
